packetbeat/protos/tls/tls_test.go

```diff
@@ -21,7 +21,7 @@ type eventStore struct {
+	expectedClientHello = "{\"dst\":{\"IP\":\"192.168.0.2\",\"Port\":27017,\"Name\":\"\",\"Cmdline\":\"\",\"Proc\":\"\"},\"server\":\"example.org\",\"src\":{\"IP\":\"192.168.0.1\",\"Port\":6512,\"Name\":\"\",\"Cmdline\":\"\",\"Proc\":\"\"},\"status\":\"Error\",\"tls\":{\"client_certificate_requested\":false,\"client_hello\":{\"extensions\":{\"_unparsed_\":[\"56026\",\"renegotiation_info\",\"23\",\"status_request\",\"18\",\"30032\",\"43690\"],\"application_layer_protocol_negotiation\":[\"h2\",\"http/1.1\"],\"ec_points_formats\":[\"uncompressed\"],\"server_name_indication\":[\"example.org\"],\"session_ticket\":\"\",\"signature_algorithms\":[\"ecdsa_secp256r1_sha256\",\"rsa_pss_sha256\",\"rsa_pkcs1_sha256\",\"ecdsa_secp384r1_sha384\",\"rsa_pss_sha384\",\"rsa_pkcs1_sha384\",\"rsa_pss_sha512\",\"rsa_pkcs1_sha512\",\"rsa_pkcs1_sha1\"],\"supported_groups\":[\"(unknown:0x6a6a)\",\"x25519\",\"secp256r1\",\"secp384r1\"]},\"supported_ciphers\":[\"(unknown:0x3a3a)\",\"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256\",\"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256\",\"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384\",\"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384\",\"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256\",\"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256\",\"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA\",\"TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA\",\"TLS_RSA_WITH_AES_128_GCM_SHA256\",\"TLS_RSA_WITH_AES_256_GCM_SHA384\",\"TLS_RSA_WITH_AES_128_CBC_SHA\",\"TLS_RSA_WITH_AES_256_CBC_SHA\",\"TLS_RSA_WITH_3DES_EDE_CBC_SHA\"],\"supported_compression_methods\":[\"NULL\"],\"timestamp\":\"1997-05-01T00:11:26Z\",\"version\":\"3.3\"},\"handshake_completed\":false,\"resumed\":false},\"type\":\"tls\"}"
```

Comment:
FYI you can use a back quotes to avoid having to escape the quotes. https://golang.org/ref/spec#String_literals

Reply:
thanks!